assets/src/edit-story/karma/fixture/containers/designPanel/videoPoster.js

```diff
@@ -48,4 +48,8 @@ export class VideoPoster extends AbstractPanel {
       name: /reset/i,
     });
   }
+
+  get panelTitle() {
+    return this.getByRole('button', { name: 'Poster' });
+  }
```

Comment:
There is a yet [unimplemented](https://github.com/google/web-stories-wp/blob/7e6e9e0a90ba301b567b5efe4674c5338d26011c/assets/src/edit-story/karma/fixture/containers/designPanel/abstractPanel.js#L31-L34) `get title() {}` in `AbstractPanel`, but it would require passing the panel title in the constructor - something like changing the `super` call in the video poster panel constructor to `super(node, path, 'Poster');` and storing that argument locally in `AbstractPanel`. And similarly for all the other panels.

Or you can set it from `DesignPanel`, where the title is already used to find the individual panels to avoid duplicating it - e.g. something like:

```js
  get videoPoster() {
    const title = 'Poster';
    const panel = this._get(
      this.getByRole('region', { name: title }),
      'poster',
      VideoPoster
    );
    panel._title = title;
    return panel;
  }
```

It's just a _nice-to-have_ if you can find a clean solution for it.

Reply:
Looked a bit into this but I think I'll do this the next time it becomes relevant to a PR (or if there's time left from the issues of this sprint). Thanks for bringing this up!